src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+* {
+	font-family: 'Robot';
+	font-size: 12px;
```

Comment:
move font family and font size to html selector
Robot => Roboto
Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)